### PR TITLE
Added typecasting suffixes to README

### PR DESCRIPTION
**Pull request by matthdsm**
I also believe PR's are better ;)

M

### CODE REVIEW

**Comment by matthdsm** [comment]
There we go, that should do the trick!

File: README.md [hunk 1]
```diff
@@ -82,6 +82,14 @@ conf file as desired. Files can be local as above, or available via http/https.
 
 Also see the additional usage section at the bottom for additional details.
 
+Typecasting values
+------------------
+
+It's possible to add field type info to the field name. By default, all fields will have `type=String`.
```
Comment (brentp) [inline README.md]: Nitpick but how about this:
By default, using op of mean`,`max`,`sum`, `div2` or `min` will result in `type=Float`, using `self` will get the type from the annotation VCF and other fields will have `type=String`

and drop the final sentence from this changeset.

Comment (matthdsm) [inline README.md]: Done!

**Comment by brentp** [comment]
cheers.

**Comment by matthdsm** [comment]
My pleasure!

M

Op 2 feb. 2017 om 15:29 heeft Brent Pedersen - Bioinformatics <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


cheers.

-

**Pull request merged**